plugins/package-managers/bazel/src/main/kotlin/Bazel.kt

```diff
@@ -164,11 +164,10 @@ class Bazel(
         }.getOrNull()
 
     private fun getDependencyGraph(projectDir: File, depDirectives: Map<String, BazelDepDirective>): Set<Scope> {
-        val modGraphProcess = run(
+        val node = run(
             "mod", "graph", "--output", "json", "--disk_cache=", workingDir = projectDir
-        )
+        ).stdout.parseBazelModule()
```

Comment:
I believe it to be good practice here to keep the `ProcessCapture` instance in a dedicated variable, just in case other properties like `stderr` will be used as well in the future.